### PR TITLE
update display of events on map page

### PR DESCRIPTION
**Pull request by lexidelorey**

### CODE REVIEW

File: src/main/webapp/mapScript.js [hunk 1]
```diff
@@ -183,7 +183,28 @@ function getUserPosts() {
   }
   return userPostDivElement;
 }
+/**
+  Get all public events to display on map page even when user isn't logged in
+ */
+
```
Comment (usaeedw) [inline src/main/webapp/mapScript.js]: Nit: Please remove empty line.

File: src/main/webapp/mapScript.js [hunk 2]
```diff
@@ -183,7 +183,28 @@ function getUserPosts() {
...
+function getPublicEvents() {
+  eventDivElement = document.createElement("div");
+  eventDivElement.innerText = '';
+  locationName = sessionStorage.getItem('locationName');
 
+  fetch("events")
+    .then(response => response.json())
+    .then(events => {
+      for (i = 0; i < events.length; i++) {
+        if (events[i].location == locationName) {
```
Comment (usaeedw) [inline src/main/webapp/mapScript.js]: I recommend combining these two conditions into one if-statement.

File: src/main/webapp/mapScript.js [hunk 3]
```diff
@@ -183,7 +183,28 @@ function getUserPosts() {
   }
   return userPostDivElement;
 }
+/**
```
Comment (usaeedw) [inline src/main/webapp/mapScript.js:187]: Nit: Recommend adding a line.

File: src/main/webapp/mapScript.js [hunk 4]
```diff
@@ -183,7 +183,28 @@ function getUserPosts() {
...
+        if (events[i].location == locationName) {
+          if (events[i].privacy == "public") {
+            eventDivElement.appendChild(createEvent(events[i]));
+          }
+        }
+      }
+    });
+  return eventDivElement;
+}
 /**
```
Comment (usaeedw) [inline src/main/webapp/mapScript.js:208]: Add a line between the end of a function and the beginning of the next one.

**Review by usaeedw — DISMISSED** [review]

**Review by ellisvivian — APPROVED** [review]

**Review by Omarzintan — APPROVED** [review]

**Review by Omarzintan — APPROVED** [review]

**Pull request merged**